pkgs/top-level/all-packages.nix

```diff
@@ -11765,9 +11765,10 @@ let
 
   xdotool = callPackage ../tools/X11/xdotool { };
 
-  xen = callPackage ../applications/virtualization/xen {
-    stdenv = overrideGCC stdenv gcc45;
-  };
+  xen_4_4_1 = callPackage ../applications/virtualization/xen/4.4.1.nix { };
+  xen_4_5_0 = callPackage ../applications/virtualization/xen/4.5.0.nix { };
+  xen_xenServer = callPackage ../applications/virtualization/xen/4.5.0.nix { xenserverPatched = true; };
```

Comment:
I don't know enough about Xen and what the patches from the XenServer patch queue do, so I can't say.
I just thought it would make sense to keep Xen 4.5.0 as it was released, like Xen 4.4.1 as well.